web/src/pages/Docs.vue

```diff
@@ -20,18 +20,23 @@
       </md-field>
     </template>
     <iframe id="docs" :src="docURL" @load="onChange()"></iframe>
+    <ClaimButton class="docs-claim-button" />
+    <DeleteButton class="docs-delete-button" />
   </Layout>
 </template>
 
 <script>
 import Layout from '@/components/Layout.vue'
-
 import ProjectRepository from '@/repositories/ProjectRepository'
+import ClaimButton from '@/components/ClaimButton.vue'
```

Comment:
I would not add the buttons to the docs view, this is confusing as one would expect to delete the current docs, wich is not the case.